STYLEGUIDE.md

```diff
@@ -2,70 +2,74 @@
 
-As a developer-focused product, PostHog depends on its documentation to provide a good experience to users who wish to use our product. Thus, it is important that our Documentation is complete, concise, and well-written.
-
-Hence, this guide aims to provide some basic guidelines for those wishing to contribute to the Documentation.
+PostHog is a product for developers, so our documentation is key to a good developer experience. We want it to be complete and concise, well-written, and easy to understand.
 
 ## Guidelines
 
-### Golden rule: assume (almost) nothing
+### Golden rule: assume almost nothing
+
+You likely know more than your reader in the area you're writing about. That is why you're documenting it!
 
-Assume as little as possible about your reader regarding the topic at hand. Whenever possible, you should provide links to other Docs and pages which describe how to download, update, and debug certain tools, so that your reader can easily solve an issue without having to go out searching on their own.
+As you gain mastery of a product or feature, some things become second nature. You know them without thinking about it. But they weren't so obvious at first. Call these out, and provide links to relevant docs or websites.
 
-Regarding this point, there is a tradeoff. You do not want your text to get too long by teaching your reader how to turn on their laptop (they hopefully have that down), but you do want to instruct them on anything that is **essential** for completing a certain action. 
+Make it easy for your reader to implement their feature or solve their issue, whether they are an expert or just starting out with PostHog.
 
-It is dangerous to assume that because something is obvious to you, it will also be obvious to your reader - just keep that in mind.
+Of course our audience is developers, so you can assume _some_ baseline knowledge.
 
 ### Additional suggestions
 
 **Use American English**
 
-PostHog is a a global company, with a team distributed across the world. As a result, to keep our communication consistent, we use American spelling for our Documentation.
+PostHog is a global company. Our team and our customers are distributed around the world. For consistency, we use American English spelling and grammar.
 
-**Use the Oxford comma for lists** 
+**Use the Oxford comma** 
 
-Example: Write "bananas, apples, and oranges", not "bananas, apples and oranges".
+Write "bananas, apples, and oranges", not "bananas, apples and oranges".
 
-**Capitalize the names of tools, protocols, and technologies**
+Why does this matter? Consider the old joke:
 
-Example: Write "Redis server", not "redis server".
+"There are two hard problems in computer science: naming things, cache invalidation, and off by one errors."
 
-**Capitalize acronyms**
+That doesn't work without the Oxford. Of course we are not so fussy in real life, but we are persnickity with our documentation.
```

Comment:
persnickity - I'm British so I was expecting this to be pernickity 😄  But we do use American English.

Since I searched, I also noticed Google suggests spelling with an "e": pernickety.

Reply:
You like "to-MAY-to", I like "to-MAH-to", [let's call the whole thing off](https://en.wikipedia.org/wiki/Let%27s_Call_the_Whole_Thing_Off)!

But seriously, it's nice to meet a colleague who cares about these details and has a sense of humor about them! I grew up in Oregon, so people think I have a Canadian accent, and that's almost British, isn't it, eh?